sql/src/test/java/org/opensearch/sql/sql/antlr/SQLSyntaxParserTest.java

```diff
@@ -325,6 +325,24 @@ public void can_parse_query_string_relevance_function() {
             + "operator='AND', tie_breaker=0.3, type = \"most_fields\", fuzziness = 4)"));
   }
 
+
+  @Test
+  public void can_parse_query_relevance_function() {
+    assertNotNull(parser.parse(
+            "SELECT id FROM test WHERE query('address:query')"));
+    assertNotNull(parser.parse(
+            "SELECT id FROM test WHERE query('address:query OR notes:query')"));
+    assertNotNull(parser.parse(
+            "SELECT id FROM test WHERE query(\"address:query\")"));
+    assertNotNull(parser.parse(
+            "SELECT id FROM test WHERE query(\"address:query OR notes:query\")"));
+    assertNotNull(parser.parse(
+            "SELECT id FROM test WHERE query(`address:query`)"));
+    assertNotNull(parser.parse(
+            "SELECT id FROM test WHERE query(`address:query OR notes:query`)"));
```

Comment:
should we add a test with the wildcard `*` field name as valid syntax?

Reply:
Fixed in f2d77c634b1869229cbb3aa2f6952403d26fe30c